src/Symfony/Component/Workflow/DefinitionBuilder.php

```diff
@@ -27,8 +27,8 @@ class DefinitionBuilder
     private $initialPlace;
 
     /**
-     * @param string[]     $places
-     * @param Transition[] $transitions
+     * @param string[]             $places
+     * @param (Transition|array)[] $transitions
```

Comment:
I think this deserves some explanation about how such an array must be composed.